pom.xml

```diff
@@ -8,13 +8,13 @@
     <version>2.1.6.RELEASE</version>
   </parent>
 
-  <groupId>com.github.miwurster</groupId>
+  <groupId>io.github.mbigger</groupId>
```

Comment:
It shouldn't be changed for the PR. I cannot merge this in this state as I only can use my registered groupId to push to Maven Central.